.rubocop.yml

```diff
@@ -108,6 +108,7 @@ RSpec/MultipleExpectations:
   Max: 5
   Exclude:
     - "spec/mailers/**/*"
+    - "spec/views/**/*"
```

Comment:
Mailers and views usually have a lot of expectations so I think it makes sense to disable it there. Otherwise we would need start using `        expect(rendered_component).to eq(
          <<~HTML ...content.. HTML`